### PR TITLE
Star masks

### LINKED ISSUE #43: bright star masks
For now we could perhaps adopt ones from DES, but they will I think be generally different for LSST.

For one thing the are obliterated by a star will be larger for LSST

### LINKED ISSUE #44: bleed trail masks
Can maybe use DES ones for now, but they will differ for LSST

### PR DESCRIPTION
**Pull request by esheldon**
closes #43 
closes #44 

At this point there are heuristics for star radii and bleed trail properties, but we can alter it to take in those values if we get a catalog of these from DC2

I defined bits for these, there are no bits in the official lsst code

TODO is to add this to the simple_sim code with keywords star bool and star_kws

### CODE REVIEW

File: descwl_shear_sims/lsst_bits.py [hunk 1]
```diff
@@ -6,6 +6,11 @@
 COSMIC_RAY = np.int64(2**3)
 EDGE = np.int64(2**4)
 
+# these are not official values, as far as I know
+# the stack doesn't mark such things (ES 2020-02-18)
+STAR = np.int64(2**10)
+BLEED = np.int64(2**11)
```
Comment (beckermr) [inline descwl_shear_sims/lsst_bits.py]: Do we want to go higher here so things don't break later?

File: descwl_shear_sims/tests/test_gen_star_masks.py [hunk 2]
```diff
@@ -0,0 +1,82 @@
+"""
+copy-paste from my (beckermr) personal code here
+https://github.com/beckermr/metadetect-coadding-sims
+"""
```
Comment (beckermr) [inline descwl_shear_sims/tests/test_gen_star_masks.py]: This comment is wrong.

File: descwl_shear_sims/tests/test_gen_star_masks.py [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+import numpy as np
+
+from ..gen_star_masks import StarMasks
+from ..lsst_bits import STAR, BLEED
+from ..simple_sim import Sim
+
+
```
Comment (beckermr) [inline descwl_shear_sims/tests/test_gen_star_masks.py]: We should add a test here that the code is repeatable if we call it twice with the same rng seed.

File: descwl_shear_sims/lsst_bits.py (hunk 1 above)
Comment (esheldon) [reply descwl_shear_sims/lsst_bits.py]: We will definitely be changing these so it doesn't matter too much.

It looks like the stack masks are 32 bit so we could use 29 and 30

File: descwl_shear_sims/tests/test_gen_star_masks.py (hunk 3 above)
Comment (esheldon) [reply descwl_shear_sims/tests/test_gen_star_masks.py]: added

File: descwl_shear_sims/tests/test_gen_star_masks.py (hunk 2 above)
Comment (esheldon) [reply descwl_shear_sims/tests/test_gen_star_masks.py]: removed

File: descwl_shear_sims/gen_star_masks.py [hunk 4]
```diff
@@ -26,24 +26,25 @@ class StarMasks(object):
     center_dec: float
         Stars will be generated in a disk around this point
     radius_degrees: float
-        Stars will be in a disk with this radius
+        Stars will be generated in a disk with this radius around the coadd
+        center, default 5 degrees
     density: float
-        Number of saturated stars per square degree
-    pixel_scale: float
-        pixel scale in arcsec/pixel
```
Comment (beckermr) [inline descwl_shear_sims/gen_star_masks.py]: Are we removing this option? I don't see the docs added back.

**Pull request merged**